src/System.Globalization.Calendars/tests/GregorianCalendar/GregorianCalendarEras.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using Xunit;
+
+namespace System.Globalization.CalendarsTests
+{
+    public class GregorianCalendarEras
+    {
+        private static readonly int[] s_ADEra = new int[]
+        {
+            1
+        };
+
+        #region Positive tests
+        // PosTest1: Get the eras of Gregorian calendar
+        [Fact]
+        public void PosTest1()
```

Comment:
(The same goes for lots of the other tests in this PR, but I'll only mention it here.)